manifests/params.pp

```diff
@@ -61,7 +55,7 @@
     'framework.ssh.keypath'     => '/var/lib/rundeck/.ssh/id_rsa',
     'framework.ssh.user'        => 'rundeck',
     'framework.ssh.timeout'     => '0',
-    'rundeck.server.uuid'       => $rdeck_uuid,
+    'rundeck.server.uuid'       => fqdn_uuid($::fqdn),
```

Comment:
can you please use `$facts['fqdn']` instead?

Reply:
@bastelfreak I fixed it here and in a few other places.